pkg/console/subresource/configmap/configmap.go

```diff
@@ -77,11 +76,12 @@ func DefaultConfigMap(operatorConfig *operatorv1.Console, consoleConfig *configv
 	configMap := Stub()
 	configMap.Data = map[string]string{}
 	unsupportedRaw := operatorConfig.Spec.UnsupportedConfigOverrides.Raw
+	newConfig := getDataFromConfigMapAsYaml(managedConfig)
```

Comment:
If the managed config `404`'d, what happens here?

Reply:
I did test it; the theory would be an empty byte array is returned and then gets ignored by the mergeProcessConfig

Reply:
Tested this again with 1) regular values, 2) cm but with no data, 3) no cm in the openshift-config-managed ns.  All passed and I did not see errors in the log.